keymap_drawer/parse.py

```diff
@@ -246,8 +246,13 @@ def mapped(key: str) -> LayoutKey:
             case [ref, par] if ref in self.sticky_keys:
                 l_key = self._str_to_key(f"{self.sticky_keys[ref][0]} {par}", current_layer, key_positions)
                 return LayoutKey(tap=l_key.tap, hold=self.cfg.sticky_label, shifted=l_key.shifted)
-            case [("&out" | "&bt" | "&ext_power" | "&rgb_ug"), *pars]:
-                return LayoutKey(tap=" ".join(pars).replace("_", " ").replace(" SEL ", " "))
```

Comment:
Can you move `.replace(" SEL ", " ")` to the default processing in line 228? This way we would preserve the current behavior. (You'll probably need to run `black` after.)

Reply:
Will do!

Reply:
@caksoylar done!